src/web/data/userMapData.json

```diff
@@ -677,8 +786,9 @@
     ],
     "Mexico": [
       "Centro de Estudios Superiores de Diseño",
-      "CETYS Universidad",
```

Comment:
Why was this removed?

Reply:
Because CETYS is the abbreviation for Centro de Enseñanza Técnica y Superior Universidad, which is what I added, because some of the Insitutes that were given in the institution lists, some of them were repeated requests with different names

Reply:
Ah okay, good catch.